frontend/src/pages/lobbyDetail/AfterDraftLobbyDetail.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { Column } from 'primereact/column';
+import { DataTable } from 'primereact/datatable';
+import { ColumnGroup } from 'primereact/columngroup';
+import { Row } from 'primereact/row';
+import { Heading, Layout } from '../../atoms';
+
+export const AfterDraftLobbyDetail = ({
+  profitsPerRound,
+  usersInNomination,
+}) => {
+  const { t } = useTranslation();
+
+  const roundColumn = {
+    key: 'note',
+    field: 'note',
+    style: 'p-datatable-header',
+  };
+  const cols = usersInNomination.map((lobbyUser, i) => {
+    const column = {
+      key: lobbyUser.user_id,
+      field: lobbyUser.user_id.toString(),
+      header: lobbyUser.nickname,
+    };
+    return column;
+  });
+
+  const dynamicColumnsHeaders = cols.map((col, i) => {
+    return <Column key={col.field} header={t(col.header)} />;
+  });
+
+  const headerGroup = (
+    <ColumnGroup>
+      <Row>
+        <Column header={t('Page.AfterDraftLobbyDetail.Note')} rowSpan={2} />
+        <Column
+          header={t('Page.AfterDraftLobbyDetail.Users')}
+          colSpan={usersInNomination.length}
+        />
+      </Row>
+      <Row>{dynamicColumnsHeaders}</Row>
+    </ColumnGroup>
+  );
+
+  const lastRowFooter = rowData => {
+    return {
+      'p-datatable-footer':
+        rowData.note === 'Page.AfterDraftLobbyDetail.Notes.total',
```

Comment:
Neslo by to rozlisit jinak? Jednak az se presune prekladovy kliz z BE ciste na FE, tak tohle fungova nebude..